src/elexmodel/client.py

```diff
@@ -288,7 +307,7 @@ def get_estimates(
             results_handler.add_unit_intervals(estimand, alpha_to_unit_prediction_intervals)
 
             for aggregate in results_handler.aggregates:
-                aggregate_list = sorted(list(set(["postal_code", aggregate])), key=lambda x: AGGREGATE_ORDER.index(x))
+                aggregate_list = self.get_aggregate_list(office, aggregate)
```

Comment:
yes. previously, if we passed in aggregates like `['postal_code', 'district', 'county_classification']` this would create aggregates for:
```
['postal_code'],
['postal_code', 'district'],
['postal_code', 'county_classification']
```
Now it should do:
```
['postal_code', 'district'],
['postal_code', 'district', 'county_classification']
```

Reply:
Hmm okay. This function kinda negates the thing above about a new office as it still requires the office to be in the current list of offices. If you think that's important, we can keep it, but if not, what do you think about logic that revolves around `postal_code` and `district` depending on the provided list instead of necessarily the default?

Reply:
Everything else looks good - thanks!

Reply:
I'm not 100% sure I follow. why does it negate?

Reply:
One of my comments before was to change something so that things wouldn't fail if we had an office that wasn't in the default aggregates dictionary, but they will fail in the `get_aggregate_list()` function if the office isn't there.

Reply:
oh I see. If we have an office that isn't in the list, then the code will now fail with a valueerror when I try to retrieve from the dictionary. Do you think we should handle that in a different way?

Reply:
Yeah. My preference would be to allow for offices not in the list as long as aggregates are provided and then in this function, use the list of aggregates we have rather than the default.

But if that doesn't actually work or is too annoying, then we should probably check earlier to make sure the office is in our list and throw a more useful error.

Reply:
ok, this now works if office isn't in the `DEFAULT_AGGREGATES` dictionary. To test, comment out the `P` row in `DEFAULT_AGGREGATES` and run:
```
elexmodel 2020-11-03_USA_G --office_id=P --estimands=dem --geographic_unit_type=county --pi_method=nonparametric --percent_reporting=20 --aggregates=postal_code --aggregates=unit
```
If an unknown office is passed and no aggregates are passed, then the model runs but returns nothing, which seems like expected and wanted behaviour (since we do that if office is found in `DEFAULT_AGGREGATES` and no aggregates are passed). To test this, comment out the `P` row in `DEFAULT_AGGREGATES` and add
```
aggregates = []
``` 
On line 151 (since the cli passes `postal_code` and `unit` as aggregates by default and run:
```
elexmodel 2020-11-03_USA_G --office_id=P --estimands=dem --geographic_unit_type=county --pi_method=nonparametric --percent_reporting=20 --fixed_effects=county_classification
```

Reply:
Yeah that makes sense!

Thanks for all the changes - LGTM!